custom/inddex/reports/gaps_detail.py

```diff
@@ -104,8 +104,10 @@ def rows(self):
                     (ConvFactorGaps, row.conv_factor_gap_code),
                     (FctGaps, row.fct_gap_code),
             ]:
-                if (self._food_data.selected_gap_type in [None, gap_class.slug]
-                        and gap_code != gap_class.AVAILABLE):
+                if (gap_code != gap_class.AVAILABLE and (
+                        not self._food_data.selected_gap_type
+                        or self._food_data.selected_gap_type == gap_class.slug
+                )):
```

Comment:
nit: it might be easier to read this as nested ifs
```
if gap_code != gap_class.AVAILABLE:
    if not self._food_data.selected_gap_type or self._food_data.selected_gap_type == gap_class.slug:
```
because they are dealing with different variables and makes it easier to interpret.

Reply:
true, or I could pull out the latter clause and name it - might do that if I come back to this area with some upcoming work.